cal/cal/static/js/app.js

```diff
@@ -102,33 +102,125 @@ analyticsApp.controller('TagsCtrl', function($scope, $http){
 });
 
 analyticsApp.controller('CategoriesCtrl', function($scope, $http){
-  var url = '/v1/colorcategories.json';
+  var categoryUrl = '/v1/colorcategories';
+  $scope.categories = [];
+  $scope.categoriesForChart = [];
 
-  // Generate graph data
-  $http({ method: 'GET', url: url }).
-    success(function (data) {
-      // set the data
-      $scope.categories = [];
+  // populate the categories pie chart
+  $http({ method: 'GET', url: categoryUrl + '.json' }).
+    success(function successCallback(data) {
       for (var i = 0; i < data.results.length; i++) {
         var category = data.results[i];
         $scope.categories.push({
+          id: category.id,
+          label: category.label,
+          hours: category.hours,
+          include: true
+        });
+        $scope.categoriesForChart.push({
+          id: category.id,
           label: category.label,
-          hours: category.hours
+          hours: category.hours,
+          include: true
```

Comment:
Here would be a good place to set a temporary variable representing the category javascript object instead of explicitly setting the same keys and values twice